code/game/gamemodes/scoreboard.dm

```diff
@@ -265,6 +265,14 @@
 	score["crewscore"] -= plaguepoints
 	score["arenafights"] = arena_rounds
 
+	var/transfer_total = 0
+	for(var/datum/money_account/A in all_money_accounts)
+		for(var/datum/transaction/T in A.transaction_log)
+			if(T.amount > 0)
+				continue // This way we don't track payouts or starting funds, only money transferred to terminals or between players
```

Comment:
Flow statements like these are actually considered harmful, if you'll believe some people.

Regardless, this is a weird logical structure that would make more sense as

    if(T.amount <= 0)
        transfer_total += abs(T.amount)

Which is more self-explanatory and immediately readable.